core/src/test/scala/org/ensime/indexer/FileWatcherSpec.scala

```diff
@@ -175,24 +175,26 @@ class FileWatcherSpec extends EnsimeSpec with TimeLimitedTests
   it should "detect removed parent base directory" taggedAs (Retryable) in
     withVFS { implicit vfs =>
       withTestKit { implicit tk =>
-        val parent = Files.createTempDir().canon
-        val dir = parent / "base"
-        dir.mkdirs()
-        try {
-          withClassWatcher(dir) { watcher =>
-            // would be better if this was atomic (not possible from JVM?)
-            waitForBaseRegistered(tk)
-
-            parent.tree.reverse.foreach(_.delete())
-            val createOrDelete: Fish = {
-              case r: BaseRemoved => true
-              case a: BaseAdded => true
-              case _ => false
+        withTempDirPath { tmpDir =>
+          val parent = tmpDir.toFile.canon
+          val dir = parent / "base"
+          dir.mkdirs()
+          try {
+            withClassWatcher(dir) { watcher =>
+              // would be better if this was atomic (not possible from JVM?)
+              waitForBaseRegistered(tk)
+
+              parent.tree.reverse.foreach(_.delete())
```

Comment:
perhaps tangential to this PR, but I think we now have a method on `Path` to do this